packages/freezed/example/test/diagnosticable_test.dart

```diff
@@ -65,4 +66,9 @@ void main() {
 
     expect(timeslot, isNot(isA<DiagnosticableTree>()));
   });
+
+  test('with double quotes', () {
+    final temp = const WithDoubleQuotes();
```

Comment:
Shouldn't this throw?

Reply:
We should be getting a compilation error at the moment due to the `const` & assert(false)


Could you update the test to check that an assertion error is thrown with the correct message?